### PR TITLE
fix: favorite icon and overlapping app title

### LINKED ISSUE #9026: The favourite icon makes it look like an app has been favourited when it has not
The current fav icon makes me think the app has been faved, when it has not. The fav icon should be an outlined star, not a filled star. 

image

Not fav:

image

Fav:

image

### PR DESCRIPTION
**Pull request by saumeya**
Signed-off-by: saumeya <saumeyakatyal@gmail.com>
closes #9026 , argo-ui - https://github.com/argoproj/argo-ui/issues/192

Updated icon screenshot and overlapping title fix
long-title

yellow-button-new-pod

long-title-2

long-title-3

Note on DCO:
If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [x] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [x] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] Optional. My organization is added to USERS.md.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

File: ui/src/app/applications/components/applications-list/applications-tiles.tsx [hunk 1]
```diff
@@ -98,6 +98,16 @@ export const ApplicationTiles = ({applications, syncApplication, refreshApplicat
...
+    const getAppName = (appName: string) => {
+        const maxAppNameLength = 30;
+        if (appName.length > maxAppNameLength) {
+            return (
+                <Tooltip content={appName}>
+                    <span className='applications-list__title'>{appName}</span>
+                </Tooltip>
+            );
+        } else return <span className='applications-list__title'>{appName}</span>;
+    };
```
Comment (ciiay) [inline ui/src/app/applications/components/applications-list/applications-tiles.tsx:110]: As the `if` condition is pretty simple and clear, the return value could be simplified using ternary operator as bellow
```
return appName.length > maxAppNameLength ? 
        (
            <Tooltip content={appName}>
                <span className='applications-list__title'>{appName}</span>
            </Tooltip>
        ) : (<span className='applications-list__title'>{appName}</span>);
```
Check with `make lint-ui` before push though.

**Review by ciiay — COMMENTED** [review]
Looks good to me 👍 

I'm only a little bit concerned about the margin between the app name and the right top icons from your 2nd screenshot. Could we add a little left margin to that icon group?
Also added a small code improvement comment.

File: ui/src/app/applications/components/applications-list/applications-tiles.tsx (hunk 1 above)
Comment (saumeya) [reply ui/src/app/applications/components/applications-list/applications-tiles.tsx:110]: thanks this looks better!

**Comment by saumeya** [comment]
> Looks good to me +1
> 
> I'm only a little bit concerned about the margin between the app name and the right top icons from your 2nd screenshot. Could we add a little left margin to that icon group? Also added a small code improvement comment.

That is a bootstrapped row, so this probably should look like this but I will still check if there is a way to add those ... or styles

**Comment by pasha-codefresh** [comment]
Favorite icon looks better now , but external button still has problem, I think it worth to fix it also 

Снимок экрана 2022-04-30 в 19 39 58

**Comment by saumeya** [comment]
Thanks @pasha-codefresh , fixed it - it will now look like this for a smaller screen size(first image) but will add a bit of space in normal resolution(see second image)
new-title
normal

**Comment by pasha-codefresh** [comment]
In such case it also looks weird, because looks like you still have space and you don't show the whole application name, maybe let's think about another solution?

**Comment by saumeya** [comment]
How about this?
new-titleee

**Comment by alexec** [comment]
I think that a yellow star would be correct. Not blue. Looks odd to me. Could the outlined star be a lighter gray?

**Comment by saumeya** [comment]
@alexec we already make use of a yellow star for new pods, that was the reason I matched the stars colour with link one.

**Comment by rbreeze** [comment]
I seem to remember an open issue about changing the indicator for new pods. If we do so, I agree that a yellow star makes sense for favoriting.

Maybe a blue dot to indicate new pods?

**Comment by reginapizza** [comment]
For the record I'm fine with the new pod indicator being changed to a blue circle, but the star color here seems to be a separate issue, is it not?

**Comment by saumeya** [comment]
Sounds good.
Attaching for reference - if this looks good, should I wait for the changes to new pod icon or just update it here?
yellow

File: ui/src/app/applications/components/application-pod-view/pod-view.scss [hunk 2]
```diff
@@ -147,7 +147,7 @@ $num-stats: 2;
             }
             &__star-icon {
```
Comment (reginapizza) [inline ui/src/app/applications/components/application-pod-view/pod-view.scss:148]: change className to `new-pod-icon` (or whatever you deem appropriate here) 

Can you also make the size of the blue circle smaller? Maybe something closer to this?
```
pod-view__node__pod__star-icon {
    background: none;
    color: #0DADEA ;
    display: block;
    left: 22px;
    margin: 0px;
    position: absolute;
    top: -4px;
    font-size: 10px;
}
```
(the blue color I changed it to is the same one as the ellipsis, I just thought it should be a little darker than the progressing color to distinguish it a little more from the green, they kind of blend together)
new-pod-icon

**Review by reginaelyse — CHANGES_REQUESTED** [review]

File: ui/src/app/applications/components/application-pod-view/pod-view.scss (hunk 2 above)
Comment (saumeya) [reply ui/src/app/applications/components/application-pod-view/pod-view.scss]: Thanks @reginapizza , made these changes!!

**Review by rbreeze — APPROVED** [review]
Nice! Thank you!

**Pull request merged**